applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -143,8 +142,13 @@ config BT_AUDIO_UNICAST_CLIENT
 	bool
 	default y
 
+config BT_ISO_TX_BUF_COUNT
+	int
```

Comment:
If this is just changing the default then there is no need for the `int` type.
```suggestion
```
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#common-kconfig-shorthands
> For a symbol defined in multiple locations (e.g., in a Kconfig.defconfig file in Zephyr), it is best to only give the symbol type for the “base” definition of the symbol

Reply:
Done, I removed all int, bool, string etc from this file. But the rest of the Kconfig.default files we should do as a separate PR

Reply:
>  But the rest of the Kconfig.default files we should do as a separate PR

agreed.